src/briefcase/platforms/web/static.py

```diff
@@ -96,33 +116,333 @@ def _trim_file(self, path, sentinel):
+                                )
+                            else:
+                                self.console.info(
+                                    f"    {filename}: Adding {insert} insert for {target}"
+                                )
+                                try:
+                                    text = wheel.read(filename).decode("utf-8")
+                                except UnicodeDecodeError as e:
+                                    raise BriefcaseCommandError(
+                                        f"{filename}: insert must be UTF-8 encoded"
+                                    ) from e
+
+                                contrib_key = f"{package_key} (deploy insert: {rel_inside} from {filename})"
+
+                                pkg_map = inserts.setdefault(target, {}).setdefault(
+                                    insert, {}
+                                )
+                                if package_key in pkg_map and pkg_map[package_key]:
+                                    pkg_map[package_key] += "\n" + text
+                                else:
+                                    pkg_map[package_key] = text
+
+                else:
+                    self.console.debug(
+                        f"    {filename}: skipping, not a supported insert."
                     )
-                    css_file.write(f" * {package}::{'/'.join(path.parts[2:])}\n")
-                    css_file.write(
-                        " *******************************************************/\n\n"
+
+    def extract_backend_config(self, wheels):
+        """Processes multiple wheels to gather a config.toml and a base pyscript.toml
+        file.
+
+        :param wheels: A list of wheel files to be scanned.
+        """
+        config_package = None
+        config_package_list = []
+        config_filename = None
+        pyscript_version = "2024.11.1"
+        pyscript_config = None
+
+        # Find packages containing a config.toml file.
+        for wheelfile in wheels:
+            with ZipFile(wheelfile) as wheel:
+                for filename in wheel.namelist():
+                    path = Path(filename)
+                    if (
+                        len(path.parts) == 3
+                        and path.parts[1] == "deploy"
+                        and path.name == "config.toml"
+                    ):
+                        self.console.info(f"    Found {filename}")
+                        config_package_list.append(wheelfile)
+                        config_filename = filename
+
+        # Return a blank pyscript config if no configuration file is found.
+        if len(config_package_list) == 0:
+            pyscript_config = {}
+        # Raise an error if more than one configuration file is supplied.
+        elif len(config_package_list) > 1:
+            raise BriefcaseConfigError(
+                f"""Only 1 backend configuration file can be supplied.
```

Comment:
AP style guide says `1` is always written as the word.
```suggestion
                f"""Only one backend configuration file can be supplied.
```